pkg/gvk/gvk.go

```diff
@@ -119,8 +119,8 @@ func toSet(strSlice []string) mapset.Set {
 	return mapset.NewSetFromSlice(interfaceSlice)
 }
 
-// collectResources collects all namespaced scoped apiResources from the cluster
-func collectResources(discovery discovery.DiscoveryInterface) ([]*metav1.APIResourceList, error) {
+// collectNamespacedResources collects all namespace-scoped apiResources from the cluster
+func collectNamespacedResources(discovery discovery.DiscoveryInterface) ([]*metav1.APIResourceList, error) {
```

Comment:
I made a few changes to the `collectResources` and `GetGVRsForCluster` funcs to more accurately reflect that they are only retrieving NS scoped resources.